OWNERS

```diff
@@ -1,7 +1,6 @@
 reviewers:
 - brancz
 - bwplotka
```

Comment:
Feel free to remove me from now, I was not very active the last half-year. If I will have more contributions and there will be value in having me as a maintainer going forward we can vote then. (:

Reply:
Removed :crying_cat_face: